app/assets/javascripts/_trip-drawer.js

```diff
@@ -1,12 +1,73 @@
+// Util functions
+
+function equalDouble(d1, d2) {
+  // cheeck equality between two (1 x 2) arrays
+  return (d1[0] === d2[0] && d1[1] === d2[1]);
+}
+
+function equalPointArray(a1, a2) {
+  // check n x 2 arrays for equality
+  if (a1.length === a2.length) {
+    var i = a1.length;
+    while (i--) {
+      if (!equalDouble(a1[i], a2[i])) {
+        return false;
+      }
+    }
+    return true;
+  }
+  return false;
+}
+
+function numberAppendString(num, string) {
+  if (num > 0) {
+    return num + string;
+  }
+  return "";
+}
+
+function secondsToStringFR(seconds) {
+  var numyears = Math.floor(seconds / 31536000);
+  var numdays = Math.floor((seconds % 31536000) / 86400);
+  var numhours = Math.floor(((seconds % 31536000) % 86400) / 3600);
+  var numminutes = Math.floor((((seconds % 31536000) % 86400) % 3600) / 60);
+  var numseconds = (((seconds % 31536000) % 86400) % 3600) % 60;
+  
+  var sentence = "";
+  sentence = sentence + numberAppendString(numyears, " ans ");
+  sentence = sentence + numberAppendString(numdays, " jours ");
+  sentence = sentence + numberAppendString(numhours, " heures ");
+  sentence = sentence + numberAppendString(numminutes, " minutes ");
+  
+  return sentence;
+}
+
+function metersToKmString(meters) {
```

Comment:
Might add comment
```
// Rounded to 1 decimal (decimeters)
// metersToKmString(6789) = 6.8
```

Also method name is misleading since is return a Float, not a String.
Might add `toString` at the end or rename.